concordia/static/admin/editor-preview.js

```diff
@@ -20,7 +20,7 @@
             .parents('.form-row')
             .first();
         $formRow.addClass('codemirror-with-preview');
-        $formRow.find('label').remove();
+        //$formRow.find('label').remove();
```

Comment:
If we're changing this we should delete the line so there's no question about intent

Reply:
I agree. I'll make the change.